content/pages/community/operators-rules.md

```diff
@@ -0,0 +1,111 @@
+---
+Title: XMPP Operators Channel
+Url: community/channels/operators
+Save_as: community/channels/operators.html
+Top_menu_show: false
+Top_Menu_order: -1
+Dropdown_menu_show: false
+Footer_show: false
+Sidebar_menu_show: false
+---
+
+## About the XMPP Operators Channel
+
+The XMPP Operators Channel is located at [operators@muc.xmpp.org](xmpp:operators@muc.xmpp.org?join). It is a place primarily for operators of federated XMPP services, to have civil and low-bar discourse and resolution of interoperability issues. In addition to that primary purpose, it is also a place for the wider XMPP community (including end-users and operators of non-federated servers) to get in touch with operators to resolve issues, report abuse or any other service related topic.
```

Comment:
it somehow gets not rendered - but I guess the website will do so, right?

Reply:
Yes, GitHub only renders HTTP(S) links.